Commands/ImportGA4Reports.php

```diff
@@ -75,6 +78,14 @@ protected function executeImpl(InputInterface $input, OutputInterface $output)
 
         $idSite = $this->getIdSite($input);
         LogToSingleFileProcessor::handleLogToSingleFileInCliCommand($idSite, $output);
+
+        $canProcessNow = $this->checkIfCanProcess();
+        if($canProcessNow['canProcess'] === false){
+            $exceededMessage = 'You have exceeded your available quota. Next retry at ' . $canProcessNow['nextAvailableAt'];
```

Comment:
Reminder: Update this message as discussed in the meeting

Reply:
Thanks. Updated.